modules/ding_nodelist/templates/_ding_nodelist_more_links.tpl.php

```diff
@@ -9,7 +9,7 @@
   <ul>
     <?php foreach ($links as $key => $link) : ?>
       <li>
-        <div class="more-link"> <?php print l(t($link['text']), $link['links']); ?></div>
+        <div class="more-link"> <?php print l(t('@morelink',array('@morelink' => $link['text'])), $link['links']); ?></div>
```

Comment:
Please remove empty space in `<div class="more-link"> <?php`  and add one before `array` declaration.